docs/src/main/asciidoc/rest-data-panache.adoc

```diff
@@ -132,10 +134,10 @@ It can be used in your resource interface:
 @ResourceProperties(hal = true, path = "my-people")
 public interface PeopleResource extends PanacheEntityResource<Person, Long> {
     @MethodProperties(path = "all")
-    Response list();
+    Person list(Page page, Sort sort);
 
     @MethodProperties(exposed = false)
-    void delete(Long id);
+    boolean delete(Long id);
```

Comment:
We end up returning a boolean response?

Reply:
Yes. Since I separated the `RestDataResource` implementation from a JAX-RS resource I decided to add a return value to the delete method so that a JAX-RS resource would know whether an entity was deleted or not and return either 204 or 404 response.